app/src/main/java/com/example/farmeraid/data/InventoryRepository.kt

```diff
@@ -11,9 +11,74 @@ class InventoryRepository {
         return flow {
             val inventoryList = listOf(
                 HomeModel.Produce(
-                produceName = "Test Produce",
-                produceAmount = 5,
-            ))
+                    produceName = "Test Produce sldkfjslkdfjsldkjf",
```

Comment:
Do we want meaningful names for the sake of demo?

Reply:
Yeah dont worry, this will be modified beofre demo